modules/core/server/services/influx.server.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var path = require('path'),
+    influx = require('influx'),
+    config = require(path.resolve('./config/config'));
+
+/**
+ * Get InfluxDB Client
+ */
+exports.getClient = function(callback) {
+  if (!config.influxdb ||
+      !config.influxdb.enabled ||
+      !config.influxdb.options ||
+      !config.influxdb.options.host ||
+      !config.influxdb.options.database) {
```

Comment:
Totally a personal thing, I find this style of coding confusing. When glancing at it I assumed you were checking that one value existed several levels deep rather than several variables exist.

I've started using lodash's `_.get()` in these cases. It's really nice to say `var enabled = _.get(config, 'influxdb.enabled')` and then check `!_.isUndefined(enabled)`.